### DISCUSSION TITLE
[Camera Support]: PTZ preset encoding issue

### DISCUSSION THREAD

**Discussion by FusselTV** [blakeblackshear/frigate#12730] [Camera Support]
### Describe the problem you are having

I have encountered an issue with encoding umlauts (üäö) for the PTZ camera presets. The umlauts are not being encoded correctly, as shown in the attached screenshot. I am unsure if the problem is caused by frigate or onvif.
![Screenshot from 2024-08-02 20-45-33](https://github.com/user-attachments/assets/40321649-9925-4010-9461-f7241493f714)

### Steps to reproduce

not necessary

### Version

0.14.0-7c39b17

### Frigate config file

```yaml
not necessary
```

### Relevant log output

```shell
not necessary
```

### Operating system

Proxmox

### Install method

Docker Compose

### Network connection

Wireless

### Camera make and model

reolink e1 zoom

**Comment by hawkeye217** [comment]
I don't think this is a Frigate bug. I created a preset on my Dahua PTZ named "üäö", and this is what I see in my preset list:

![Screenshot 2024-08-02 at 2 11 19 PM](https://github.com/user-attachments/assets/816d6537-cd5d-48fa-98a4-a11b3f4f887a)


Rather, this is most likely a bug in your camera firmware's ONVIF implementation. To be sure, can you provide the output of `<frigate_ip>:<port>/api/<camera name>/ptz/info`? The output of presets from this api call is what is returned from the camera itself.

**Reply by FusselTV** [reply]
You are right, that's unfortunate:

`{"features":["pt","zoom"],"name":"E1","presets":["tisch","k\u00e3\u00bcche","t\u00e3\u00bcr","fenster","privacy"]}`

Thx for the quick response

**Reply by hawkeye217** [reply]
Happy to help. See if you can file a bug with reolink!
